packages/graphql-client/.eslintrc.cjs

```diff
@@ -0,0 +1,41 @@
+module.exports = {
```

Comment:
Ideally I'd like to use the same eslint config for all packages within this repo to make them as consistent as possible. Is that an option here?

Reply:
I think so - I've removed it for now but other than just rule differences, the main thing that is different between this eslint config and the main one is that this one has `browser: true`. So far this hasn't really cause any issues but it might in the future.